src/providers/LayerProvider/LayerProvider.tsx

```diff
@@ -47,7 +49,9 @@ export const LayerProvider = ({
           appSecret={appSecret}
           businessAccessToken={businessAccessToken}
         >
-          <BusinessProvider {...restProps} />
+          <BusinessInputProvider businessId={businessId}>
```

Comment:
We need to add Categories context here ...OR add it to the BankTransactions, because categories are needed in the split/categorize form:

![image](https://github.com/user-attachments/assets/6b815cb6-ae5a-463a-a13c-0156372838b7)

I think categories can be commonly used so having it here seems to be acceptable, but still due to `useSWR` caching mechanism it's fine to use categories when it's needed.